pyproject.toml

```diff
@@ -1,2 +1,58 @@
 [tool.black]
 target-version = ['py310']
+
+[tool.poetry]
+name = "screenpy"
+version = "4.0.2"
+description = "Screenplay pattern base for Python automated UI test suites."
+authors = ["Perry Goy <perry.goy@gmail.com>"]
+license = "MIT"
+#packages = []
+repository = "https://github.com/ScreenPyHQ/screenpy"
+documentation = "https://screenpy-docs.readthedocs.io"
+readme = "README.md"
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Quality Assurance",
+    "Topic :: Software Development :: Testing",
+    "Topic :: Software Development :: Testing :: BDD",
+    "Natural Language :: English",
+    "License :: OSI Approved :: MIT License",
+]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+PyHamcrest = "~2.0"  # ">=2.0.0,<2.1"
+typing-extensions = ">=4.1.1,<4.4"
```

Comment:
Is there a reason to limit the max version for hamcrest and typing-extensions?

Reply:
I usually limit to the next minor version just in case, but maybe limiting to major versions would be safe enough. The general rule of thumb is that you bump major versions when there is significant overhaul or breaking changes, minor versions when new features are added, hotfix versions (if you use them) when there are just, y'know, hotfixes.

That's why i was limiting the top in the first place. Maybe we can just say `<5` though.

Reply:
> Maybe we can just say <5 though.
I lean toward this being the default for most package dependencies.  Assume minor future updates will be fine unless we find otherwise, in which case we put upper limits where needed.  

The flip side is we would need to bump the dependencies on a regular basis.

Reply:
I'm going to remove the top end restrictions where possible.